ansible/roles/etcd/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# tasks file for etcd
+
+- name: install etcd from centos extras repository
+  yum: name=etcd state=installed
```

Comment:
we could specify the version to be installed, to ensure everyone gets the same version?